tests/unit/specs/components/AboutMe/AboutMyPersonalInfo/AboutMyPersonalInfo.nuxt.spec.ts

```
@@ -0,0 +1,64 @@
+      expect(age.text()).toBe("25 ans");
+    });
+  });
+
+  describe("Phone Number", () => {
+    it("should render formatted phone number when rendered.", () => {
+      const phoneNumber = wrapper.find<HTMLDivElement>("#phone-number");
+
+      expect(phoneNumber.text()).toBe("12 34 56 78 90");
+    });
+
+    it.skip("should render ? when phone number in runtime config doesn't have a correct format.", async() => {
```

ESLint

[error] Disabled test - if you want to skip a test temporarily, use .todo() instead

(vitest/no-disabled-tests)